dlpy/model_conversion/write_keras_model_parm.py

```diff
@@ -35,7 +35,10 @@
 try:
     from keras.engine.topology import preprocess_weights_for_loading
 except ImportError:
-    from keras.engine.saving import preprocess_weights_for_loading
+    try:
+        from keras.engine.saving import preprocess_weights_for_loading
+    except:
```

Comment:
You should make this only catch ImportError

Reply:
Good catch - I meant to condition that on ImportError but overlooked it.  My latest update corrects this oversight.